skimage/filters/rank/generic.py

```diff
@@ -127,6 +129,30 @@ def _apply_vector_per_pixel(func, image, selem, out, mask, shift_x, shift_y,
     return out
 
 
+def default_selem(func):
```

Comment:
Please, consider merging this with https://github.com/scikit-image/scikit-image/blob/master/skimage/morphology/selem.py#L342.